packages/babel-plugin-transform-unicode-property-regex/test/fixtures/with-unicode-flag/script-zzzz/exec.js

```diff
@@ -0,0 +1,3 @@
+expect(/\p{Script=Zzzz}/u.test("\uD800")).toBe(true);
```

Comment:
We didn't include an `input.js` because `\p{Script=Zzzz}` is a superset of `\p{Unassigned}`, which is not quite reviewable and it will be updated in every Unicode releases.